modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,288 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+MuleSoft provides application logs and a runtime log to help you monitor and troubleshoot your apps and the Mule server.
+
+== App Logs
+
+Every app that you build in Studio comes with its own `log4j2.xml` file. The log contains information about any errors raised in the app (unless you have app logic to handle those errors). It also contains anything you want to explicitly log, if you build the logic in the app.
+
+Mule automatically logs multiple messages and specific elements in your app flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[Logger component] anywhere in a flow and set it up to output any message you want.
```

Comment:
I would place the xref:logger-component-reference.adoc[Logger component] in the See Also section and make the name bold for the name here.

Reply:
I could, but folk's don't look at the See Also that often. I think this component is heavily used and is an integral part of logging, so I want to keep the link to it here.